apps/cli-daemon/tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "noImplicitOverride": true,
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "resolveJsonModule": true
```

Comment:
why is it needed? I didnt see that you import json anywhere, if its for jest, move it to tsconfig.spec.json

Reply:
im importing angular.json file.